modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,18 @@
+= Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MuleSoft Government Cloud is a secure, FedRAMP-compliant deployment environment that enables government agencies to use Anypoint Platform in the cloud.
+
+// (Specialized deployment of Anypoint Platform for government agencies.)
+
+Your agency might already use Anypoint Platform on-premises to build applications. With Government Cloud, you can leverage the integration platform as a service (iPaaS) to reduce the infrastructure and management costs associated with legacy on-premises integration.
+
+With MuleSoft managing the Anypoint Platform infrastructure, your IT teams can focus on designing integrations. And, you can manage and monitor all government integration assets from a single secure, cloud-based management console.
```

Comment:
ditto re. beginning sentence with a conjunction. Not wrong, but I advise against it.

Reply:
Removed.